### DISCUSSION TITLE
Is it possible to show an existing pdf file as preview

### DISCUSSION THREAD

**Discussion by Alvi24** [diegomura/react-pdf#2788] [General]
Is it possible to show an existing pdf file as preview, to show just the first page of that pdf file as image

**Comment by farida-mullamitha** [comment]
yeah it is possible you can use react-pdf for rendering and displaying existing PDF files.

**Comment by diegomura** [comment]
Hi! No, this is not possible with this library. react-pdf only generates new PDF documents from React components. It cannot open or display an existing PDF file.

For showing an existing PDF, or its first page as an image, take a look at [react-pdf by wojtekmaj](https://github.com/wojtekmaj/react-pdf), which is a different library made exactly for that. It renders PDF pages to a canvas, so you can show just the first page as a preview.

Closing this one since it is out of scope for this library. Thanks!
